structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,125 @@
+
+### Managers
+
+Managers are responsible for keeping the collab program up and running.
+
+- Maintain and organise the associated Discord server
+- Actioning changes to the collab-related spaces
+- Organise polls and votes as required by formal processes
+- Help resolve conflicts between communities
+- Stay in touch with any community part of the program
+
+Community Collab Managers are the same as Community Managers of the Quilt Community Team.
+
+### Collaborators
+
+Collaborators are moderators of one or more communities in the program. They have access to the server to the moderation part of the server, such as general moderator chats and case threads. Access is also given to the event category.
+
+### Event Planners
+
+Event Planners are members of the staff team of one or more communities in the program, who focus on event organisation. They have access to the events part of the server.
+
+### Project Roles
+
+Other roles may be given surrounding the different projects happening inside the program. Those roles are informal and will depend on each project.
+
+### Invitees
+
+Invitees are temporary users, with no access to the rest of the server. They only have access to temporary channels required by the process they are visiting for.
+
+## Voting process
+
+If required, a vote can be called by a Manager where the involved Collab communities are each constituents. A poll and an associated thread are created in an internal channel designated for that purpose, including as many details as possible.
+
+Each community gets a single vote. They are free to write their own process for determining their vote, but it must follow those rules:
+
+- This process must be democratic and give a voice to any community member who is able to join the Community Collab server under the Collaborator role.
+- Each eligible voter must have single, equal voting power.
+- The process can remove the voting power under some reasonable inactivity threshold.
+
+Once a community has decided their vote, one member can report their vote in the thread, mentioning the community name, and the option being voted for, with corresponding proof (such as a screenshot of an internal poll).
+
+Each community also has the option of skipping its vote if none of its members has an opinion on the poll. Such instances should be reported the same way.
+
+Polls must include a closing date of at least ten days. If required, the closing date can be pushed away by a maximum of a month, upon the informal agreement with the majority of the other communities. During that period, any Collaborator can use their veto right (see below).
+
+Once the poll is closed, the vote can be interpreted, depending on the specific process this vote is part of, if any.
+
+## Veto rights
+
+While the vote is active, any Collaborator or Manager can use their veto right by sending a message inside the thread, detailing their reasoning.
```

Comment:
What happens if a Collaborator/Manager withdraws their veto? 
Can a Collaborator/Manager make multiple vetos based on different reasonings (one after another)?

My opinion is that a veto should require a detailed explanation and reasoning to be valid, and that a veto which does not provide such explanation/reasoning to be held as invalid.

Also, to clarify, each community gets one equal vote, but each Collaborator (of which a community may have many) gets one veto?

Reply:
> What happens if a Collaborator/Manager withdraws their veto?

The vote is considered failed right away, and we carry on. The same goes for any other vote, I don't think there is any point in formalizing all of those points.

> Can a Collaborator/Manager make multiple vetos based on different reasonings (one after another)?

I guess not, they should be combined together

> My opinion is that a veto should require a detailed explanation and reasoning to be valid, and that a veto which does not provide such explanation/reasoning to be held as invalid.

This will be up to the other Collaborators to enforce through overruling. I don't want to have a single person be able to decide that.

> Also, to clarify, each community gets one equal vote, but each Collaborator (of which a community may have many) gets one veto?

To simply things, yes. It doesn't change anything, to be fair.

Reply:
> The vote is considered failed right away, and we carry on. The same goes for any other vote, I don't think there is any point in formalizing all of those points.

Wait, the _vote_ is considered failed when a veto is withdrawn by the party which used it?

> This will be up to the other Collaborators to enforce through overruling. I don't want to have a single person be able to decide that.

> To simply things, yes. It doesn't change anything, to be fair.

One could argue that a community could gum up the works of a vote by having their multiple Collaborators cast their veto with any reason one after another, given that we neither enforce a limit on the number of Collaborators from a community, nor require a reason for casting a veto (which puts the burden of overruling reasonless vetoes on other Collaborators/Managers).

Then again, if a community does try this obstructionist tactic, they may find themselves at the receiving end of a removal vote afterwards. 🤔

Also, what I meant by providing an explanation/reasoning is any comprehensible reason beyond "I cast my veto", so a veto _must_ have some sort of public reasoning on why it was cast. I draw inspiration from the [OpenJDK Bylaws' part on a Veto in the section for Consensus Voting](https://openjdk.org/bylaws#veto):

> A **Veto** must be accompanied by a justification; a Veto without a justification is invalid. If a Veto is raised then the voter who raised the veto and those who support the proposed action must work together to reach a mutually-agreeable resolution. If a Veto is resolved then the voter who raised the Veto must withdraw that vote by explicitly abstaining or by casting a Yes vote.

---

Also, with regards to a Collaborator/Manager's one veto, does that mean once they cast their veto, regardless of its resolution (withdrawn, overruled), they cannot cast another veto in the same vote? Are they allowed to use their veto right multiple times in a single vote?

Reply:
> Wait, the vote is considered failed when a veto is withdrawn by the party which used it?

Sorry, that wasn't clear. If withdrawn, the veto is considered as overruled immediately, if we really need to tie it to the current process, or pretend it didn't happen, that's the same result.

> Then again, if a community does try this obstructionist tactic, they may find themselves at the receiving end of a removal vote afterwards. 

Exactly, we assume good faith from communities here. Acting in bad faith would probably mean a vote against them soon enough. 

> Also, with regards to a Collaborator/Manager's one veto, does that mean once they cast their veto, regardless of its resolution (withdrawn, overruled), they cannot cast another veto in the same vote? Are they allowed to use their veto right multiple times in a single vote?

There are no specific rules on that. If the person has two concerns, they would be better off combining them into a single veto to have better chances of a common agreement. I don't want to enforce having only a single veto, though, in case other elements come in later during the voting process and do need another veto. I think we can also assume good faith here.